dev/drivers/scripts/stats/analyses/jevs_analyses_urma_grid2obs_stats.sh

```diff
@@ -9,19 +9,23 @@
 #PBS -l select=1:ncpus=1:mem=10GB
 #PBS -l debug=true
 
+set -x
+
 export model=evs
 
 export HOMEevs=/lfs/h2/emc/vpppg/noscrub/$USER/EVS
 
 source $HOMEevs/versions/run.ver
 
+export evs_ver=v1.0.0
+evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)f1
```

Comment:
Please remove the typo at the end of line 21.

Reply:
@PerryShafran-NOAA If you can't remember off the top of your head (I know I can't), this line should be:
`evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)`

Reply:
Typo removed, and confirmed that it didn't make it to the other stats script as I did a cut 'n' paste.

Reply:
@AliciaBentley-NOAA thanks!  I knew the line; that was a legit typo.  ;)